discordrb.json

```diff
@@ -57,6 +57,18 @@
             "voiceReceive": "no",
             "voiceSend": "yes",
             "voiceStateUpdate": "yes"
+        },
+        "misc": {
+            "supportsUserLogin": "yes",
```

Comment:
Yes, it is useful for non-2FA accounts. I just don't want current or future library authors to feel obligated to support a feature that is not officially documented, or to think that they need such features in order to have a "complete" library.